components/BackButton.tsx

```diff
@@ -0,0 +1,13 @@
+'use client';
+
+import { useRouter } from "next/navigation";
+
+export default function BackButton({ route }: { route: string }) {
+	const router = useRouter();
+
+	return (
+		<button className="bg-white hover:bg-gray-100 text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded shadow" onClick={() => { router.push(`${route}`) }}>
```

Comment:
We could look into making a button component that has different variants, and we could pass the variant we want as a prop. This would make code easier to read on components that need to use buttons since the downside of using TailwindCSS is all the inline styling.

Reply:
yea I was thinking the same thing, maybe someone on the frontend can set that up